Salesforce/salesforce-events/tests/salesforce_apex_rest_api.json

```diff
@@ -0,0 +1,72 @@
+  },
+  "expected": {
+    "message": "{\"EVENT_TYPE\":\"ApexRestApi\",\"TIMESTAMP\":\"20241029112721.500\",\"REQUEST_ID\":\"5-Acr8biM2uu2UsrkBY5B-\",\"ORGANIZATION_ID\":\"11111111111111111\",\"USER_ID\":\"user123123123123\",\"RUN_TIME\":\"462\",\"CPU_TIME\":\"211\",\"URI\":\"/requests\",\"SESSION_KEY\":\"session-key12323\",\"LOGIN_KEY\":\"xxxxxxxxxxxxxx\",\"USER_TYPE\":\"Standard\",\"REQUEST_STATUS\":\"S\",\"DB_TOTAL_TIME\":\"181052355\",\"METHOD\":\"POST\",\"MEDIA_TYPE\":\"application/json\",\"STATUS_CODE\":\"200\",\"USER_AGENT\":\"9999\",\"ROWS_PROCESSED\":\"0\",\"NUMBER_FIELDS\":\"\",\"DB_BLOCKS\":\"22201502\",\"DB_CPU_TIME\":\"143\",\"REQUEST_SIZE\":\"\",\"RESPONSE_SIZE\":\"\",\"ENTITY_NAME\":\"\",\"CONNECTED_APP_ID\":\"\",\"CLIENT_NAME\":\"\",\"EXCEPTION_MESSAGE\":\"\",\"TIMESTAMP_DERIVED\":\"2024-10-29T11:27:21.500Z\",\"USER_ID_DERIVED\":\"user123123123123QAC\",\"CLIENT_IP\":\"1.2.3.4\",\"URI_ID_DERIVED\":\"\"}",
+    "event": {
+      "category": [
+        "network"
+      ],
+      "dataset": "ApexRestApi",
+      "type": [
+        "info"
+      ]
+    },
+    "@timestamp": "2034-02-21T02:48:31.272150Z",
+    "http": {
+      "request": {
+        "method": "POST"
+      },
+      "response": {
+        "mime_type": "application/json",
+        "status_code": 200
+      }
+    },
+    "organization": {
+      "id": "11111111111111111"
+    },
+    "related": {
+      "ip": [
+        "1.2.3.4"
+      ]
+    },
+    "salesforce": {
+      "login": {
+        "key": "xxxxxxxxxxxxxx"
+      },
+      "request": {
+        "id": "5-Acr8biM2uu2UsrkBY5B-",
+        "status": "S"
+      },
+      "session": {
+        "key": {
+          "id": "session-key12323"
+        }
+      },
+      "user": {
+        "type": "Standard"
+      }
+    },
+    "source": {
+      "address": "1.2.3.4",
+      "ip": "1.2.3.4"
+    },
+    "url": {
+      "path": "/requests"
+    },
+    "user": {
+      "id": "user123123123123"
+    },
+    "user_agent": {
+      "device": {
+        "name": "Other"
+      },
+      "name": "Other",
+      "original": "9999",
+      "os": {
+        "name": "Other"
+      }
+    }
```

Comment:
I think we should remove `user_agent.original`, as it expect a string, and move the value to a custom field.